src/app.js

```diff
@@ -0,0 +1,83 @@
+const todoArr = [
+  {
+    description: 'Wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Manage all your lists in one place',
+    completed: false,
+    index: 2,
+  },
+];
```

Comment:
Maybe you could remove the hardcoded objects and make the array empty instead.

Reply:
Thank you for the review. I will implement the changes.